images/poetry-python3.10/context/Dockerfile

```diff
@@ -37,3 +37,5 @@ RUN python3 -m venv $POETRY_HOME && \
         --use-pep517 \
         --requirement requirements-poetry.txt && \
     rm -rf requirements-poetry.txt
+
```

Comment:
Maybe a bit nit picky, but can't we rather add this to requirements-poetry? won't that have the same effect? I would prefer that as then we can get dependabot and locking.

Reply:
It can be, that I can build this into the requirements, but I am not sure how poetry resolves this, so I am going with a recommended approach.

Reply:
But this recommended approach goes against our recommended approach, because then it is no longer reproducable

Reply:
Who recommends this approach?

Reply:
> It can be, that I can build this into the requirements, but I am not sure how poetry resolves this

Poetry has only so many places it can take things form, and it really is limited to in the (v)env that contains poetry, or the venv that poetry creates.

Reply:
I just checked, if I install poetry in a venv, and install something with poetry self add, it goes into that same venv

Reply:
Nice, then I will add it to the requirements-poetry.txt

Reply:
Updated!